cybersixgill_actionable_alerts/README.md

```diff
@@ -0,0 +1,67 @@
+# Agent Check: cybersixgill_actionable_alerts
+
+## Overview
+By integrating Cybersixgill actionable alerts, Datadog customers gain a premium,
+automated threat intelligence solution based on the most comprehensive data sources from the deep, dark and surface web. 
+It is customizable, enabling users to define key assets relevant to their brand, industry, and geolocation. Users can covertly 
+monitor critical assets such as IP addresses, domains, vulnerabilities, and VIPs for activity on the underground and closed sources - and 
+prioritize, as well as respond to threats directly from the Siemplify dashboard.
+This check monitors [cybersixgill_actionable_alerts][1].
+
+## Setup
+
+### Installation
+
+To install the cybersixgill_actionable_alerts check on your host:
+
+
+1. Install the [developer toolkit][11] on any machine.
+
+2. To build the package, run the command: `ddev release build cybersixgill_actionable_alerts` 
+
+3. [Install the Datadog Agent][10] on your host.
+
+4. Once the Agent is installed, upload the build artifact by running the command: `datadog-agent integration install -w
+ path/to/cybersixgill_actionable_alerts/dist/datadog_cybersixgill_actionable_alerts-0.0.1-py3-none-any.whl`.
+
+### Configuration
+
+1. Provide Client Id and Client Secret in Configuration.yaml file which you will get it info@cybersixgill.com2.
```

Comment:
How does the customer get the client id and client secret from the email address?

Reply:
They will get client id and client secret once they contact Cybersixgill Support team.